### DISCUSSION TITLE
我今天调查了一个调查点才发现

### DISCUSSION THREAD

**Discussion by Tartaglia770** [share121/inter-knot#2969] [其他]
![Screenshot_20240818_125821_com miHoYo Yuanshen](https://github.com/user-attachments/assets/6a5b7dcd-c471-48e5-9906-88665adca11c)
原来赤沙岩板有七级，我一直以为只有6级

**Comment by Legendary-agent** [comment]
【解答】赤沙石板第七级是沙漠最后一片区域开放的时候一起开放的，不做任务基本用不到。
